### ISSUE TITLE
Support default Resources.resw

### ISSUE THREAD

**Issue by AndrewKeepCoding** [AndrewKeepCoding/WinUI3Localizer#50]
- It will be referenced if a key is not found in the current language **Resources.resw**.
- It could be located directly in the **Strings** folder.

**Comment by beeradmoore** [comment]
We had a similar thing to what I think you are proposing here in our Xamarin.Forms app with resx. We'd set the default language and then set the users preferred language. That way if we added a new label that wasn't translated in time it'd at least fall back to default instead of blank which is what I am seeing in the out of the box WinUI3 experience.

**Comment by AndrewKeepCoding** [comment]
Yes, that's the idea. This approach will also cover cases for strings that you might not need to localize (e.g. On/Off, units, etc.).🙂
